snowpack/src/build/import-resolver.ts

```diff
@@ -10,6 +10,7 @@ import {
   replaceExtension,
 } from '../util';
 import {getUrlsForFile} from './file-urls';
+import fastGlob from 'fast-glob';
```

Comment:
I’d recommend using `glob` in this PR.

I’d be +1 for replacing `glob` with `fast-glob` everywhere. But let’s do it all in a separate PR where that’s the only goal, and we just update everything in one move.